index.js

```diff
@@ -11,7 +11,10 @@ const mongoHost = process.env.MONGODB_HOST || 'localhost'
 
 mongoose.connect(`mongodb://${mongoHost}/todo`)
 mongoose.connection.on('error', () => {
-  console.log('ERROR: Unable to connect to MongoDB.')
+  console.log('Unable to connect to MongoDB... retrying')
```

Comment:
`Unable to connect to MongoDB...` still sounds harsh? `Waiting for mongo connection`